app/models/patient.rb

```diff
@@ -900,12 +953,25 @@ def self.from_fhir(patient, default_jurisdiction_id)
       native_hawaiian_or_other_pacific_islander: PatientHelper.race_code?(patient, '2076-8'),
       ethnicity: PatientHelper.ethnicity(patient),
       sex: PatientHelper.birthsex(patient),
-      preferred_contact_method: PatientHelper.from_preferred_contact_method_extension(patient),
-      preferred_contact_time: PatientHelper.from_preferred_contact_time_extension(patient),
-      symptom_onset: PatientHelper.from_symptom_onset_date_extension(patient),
-      last_date_of_exposure: PatientHelper.from_last_exposure_date_extension(patient),
+      preferred_contact_method: PatientHelper.from_string_extension(patient, 'preferred-contact-method'),
+      preferred_contact_time: PatientHelper.from_string_extension(patient, 'preferred-contact-time'),
+      symptom_onset: PatientHelper.from_date_extension(patient, 'symptom-onset-date'),
+      last_date_of_exposure: PatientHelper.from_date_extension(patient, 'last-exposure-date'),
       isolation: PatientHelper.from_isolation_extension(patient),
-      jurisdiction_id: PatientHelper.from_full_assigned_jurisdiction_path_extension(patient, default_jurisdiction_id)
+      jurisdiction_id: PatientHelper.from_full_assigned_jurisdiction_path_extension(patient, default_jurisdiction_id),
+      monitoring_plan: PatientHelper.from_string_extension(patient, 'monitoring-plan'),
+      assigned_user: PatientHelper.from_string_extension(patient, 'assigned-user'),
+      additional_planned_travel_start_date: PatientHelper.from_date_extension(patient, 'additional-planned-travel-start-date'),
+      port_of_origin: PatientHelper.from_string_extension(patient, 'port-of-origin'),
+      date_of_departure: PatientHelper.from_date_extension(patient, 'departure-date'),
```

Comment:
For this I was just following the precedent already set by the `"last-exposure-date"` which corresponds to Last Date of Exposure, for the sake of consistency among our FHIR extensions. Not sure if there was any real reason for changing the wording like that orginially. So it's now a question of whether the consistency with the import/UI is worth the modification to the existing API. It would be easy enough to make it backwards compatible. This is the kind of small thing I can easily spend way too long just on the fence debating without coming to a conclusion. What's your opinion?

Reply:
Well if you're looking for people who won't spend way too long debating something like this, you're in the wrong place. But I would vote for consistency with import/UI. I think that makes it as obvious as possible to people which UI fields map to which FHIR. I think @holmesie originally created this though, so if he has a preference for the "date" being at the end of the extension name, I'm happy to rescind my vote.

Reply:
I don't think I had a particular reason for `last-exposure-date`, and am certainly not attached to it. I'm fine with moving towards a better style, as long as we can keep an alias or something for this particular one since folks have been using it.

Reply:
Addressed in https://github.com/SaraAlert/SaraAlert/pull/521/commits/522b0504ca5c58333ecf9a92846f8ff629269478. I opted for `last-date-of-exposure` and backwards compatibility. Will update in the IG as well.